test_fuzz_comps.py

```diff
@@ -240,12 +240,12 @@ def assigns(draw):
 
 def statements():
     return st.one_of(
+        st.from_type(ast.Name),
```

Comment:
FYI this is incorrect (Carl already figured this out in #8), it needs to be wrapped in an `Expr` node as a `Name` by itself can't be a statement.